tests/src/analysis/testqgsmeshcalculator.cpp

```diff
@@ -316,5 +320,117 @@ void TestQgsMeshCalculator::calcAndSave()
   QVERIFY( fileInfo.exists() );
 }
 
+void TestQgsMeshCalculator::onTheFlyDatasetGroup()
```

Comment:
probably worth of adding some more tests to go through more possibilites... like aggregate?